getting-started/kubearmor_vm.md

```diff
@@ -4,10 +4,11 @@ This recipe explains how to use KubeArmor directly on VM/Bare-Metal host and was
 
 ## Download and Install KubeArmor
 
-1. Install latest bcc from [source](https://github.com/iovisor/bcc/blob/master/INSTALL.md#source), based on the linux distribution you are using.`version 0.21.0 or greater`
+1. Download the [latest release of KubeArmor](https://github.com/kubearmor/KubeArmor/releases)
+2. Install KubeArmor `sudo apt install ./kubearmor_${VER}_linux-amd64.deb` ... where VER is the kubearmor release version. This will automatically install the required dependencies.
 
-2. Download the [latest release of KubeArmor](https://github.com/kubearmor/KubeArmor/releases)
-3. Install KubeArmor `sudo apt install ./kubearmor_${VER}_linux-amd64.deb` ... where VER is the kubearmor release version. This will automatically install the required dependencies.
+##### Note:
+We automatically install `bpfcc-tools` with our package, your distribution might have an older version of BCC so consider installing BCC from [source](https://github.com/iovisor/bcc/blob/master/INSTALL.md#source) in case of errors.
```

Comment:
```suggestion
> Note: We automatically install `bpfcc-tools` with our package, your distribution might have an older version of BCC so consider installing BCC from [source](https://github.com/iovisor/bcc/blob/master/INSTALL.md#source) in case of errors.
```
Let's use a blockquote